### ISSUE TITLE
Autocomplete Turn-off & Alternative List

### ISSUE THREAD

**Issue by elijahmanor** [cezarsa/silver_bird#135]
The autocomplete feature isn't very useful for me now because it uses all my followers.

It would be nice if I could rather pick a Twitter List to use instead for the autocomplete.

Also, having the ability to turn off the autocomplete would be nice too.

**Comment by cezarsa** [comment]
You're right. Actually I shouldn't have published the autocomplete feature in this version, it still needs some love and definitely a way to turn it off.
That said, you're probably seeing autocomplete suggestions for the people you're following, not for your followers, right?
I'll work on making the autocomplete list smarter, maybe basing the list on currently shown tweets would be a good idea, what do you think?

**Comment by elijahmanor** [comment]
Yeah, the autocomplete is killing my experience. I'll pull the master. 

The currently shown tweets would get some of the people you want, but probably not mostly what you want. 

Would be nice to select "followers", "friends", or "list" as the source of autocomplete.

If you had a smart version I would think maybe it would keep track of usernames that scroll through your tabs (and caches them in local storage). That way when they fall off the tab, they can be used at a later point in time.
